src/Frame.js

```diff
@@ -0,0 +1,15 @@
+"use strict";
+
+function Frame() {
+    this._DEFAULT_PINS = 10;
+    this._NULL_SCORE = null;
+    this._currentPins = this._DEFAULT_PINS;
+    this._isStrike = false;
+    this._score1 = this._NULL_SCORE;
+    this._score2 = this._NULL_SCORE;
+    this._tot_score = (this._score1 + this._score2);
```

Comment:
this seems like a strange default, particularly as `null + null` is unexpected.  With derived values like this it's often better to derive them on the fly rather than having two properties storing two versions of the truth.